PWGHF/Tasks/taskMcValidation.cxx

```diff
@@ -67,21 +68,32 @@ enum DecayChannels { DzeroToKPi = 0,
                      nChannels
 }; // always keep nChannels at the end
 
-static constexpr int nMesonChannels = 10; // number of meson channels
-static constexpr int nOriginTypes = 2;   // number of origin types (prompt, non-prompt)
-static constexpr std::array<int, nChannels> PDGArrayParticle = {o2::constants::physics::Pdg::kD0, o2::constants::physics::Pdg::kDStar, o2::constants::physics::Pdg::kDPlus, o2::constants::physics::Pdg::kDPlus, o2::constants::physics::Pdg::kDS, o2::constants::physics::Pdg::kDS, o2::constants::physics::Pdg::kDS1, o2::constants::physics::Pdg::kDS2Star,
-                                                                o2::constants::physics::Pdg::kD10, o2::constants::physics::Pdg::kD2Star0, o2::constants::physics::Pdg::kLambdaCPlus, o2::constants::physics::Pdg::kLambdaCPlus, o2::constants::physics::Pdg::kXiCPlus, o2::constants::physics::Pdg::kXiCPlus, o2::constants::physics::Pdg::kXiC0, o2::constants::physics::Pdg::kOmegaC0, o2::constants::physics::Pdg::kOmegaC0};
-static constexpr std::array<unsigned int, nChannels> nDaughters = {2, 3, 3, 3, 3, 3, 5, 5, 4, 4, 3, 3, 3, 5, 4, 4, 4};
-static constexpr std::array<int, nChannels> maxDepthForSearch = {1, 2, 2, 2, 2, 2, 3, 3, 3, 3, 2, 3, 2, 4, 3, 3, 3};
+static constexpr int nCharmMesonChannels = 10;                                                      // number of charm meson channels
+static constexpr int nBeautyMesonChannels = 1;                                                      // number of beauty meson channels
```

Comment:
For the beauty hadrons, I would not separate between mesons and baryons since the number of channels is small (we will probably have only Lb as beauty baryon)